AsyncDisplayKit/Details/ASCollectionDataController.mm

```diff
@@ -143,15 +145,14 @@ - (void)prepareForInsertRowsAtIndexPaths:(NSArray<NSIndexPath *> *)indexPaths
 
 - (void)willInsertRowsAtIndexPaths:(NSArray<NSIndexPath *> *)indexPaths
 {
-  NSArray *keys = _pendingContexts.allKeys;
-  for (NSString *kind in keys) {
+  for (NSString *kind in _pendingContexts) {
     NSMutableArray<ASIndexedNodeContext *> *contexts = _pendingContexts[kind];
```

Comment:
There should be a possible answer: https://www.objc.io/issues/7-foundation/collections/#enumeration-and-higher-order-messaging-2